msal/src/main/java/com/microsoft/identity/client/INativeAuthPublicClientApplication.kt

```diff
@@ -23,9 +23,12 @@
 package com.microsoft.identity.client
 
 import com.microsoft.identity.client.exception.MsalException
+import com.microsoft.identity.client.statemachine.results.SignInResult
+import com.microsoft.identity.client.statemachine.results.SignInUsingPasswordResult
+import com.microsoft.identity.client.statemachine.states.AccountResult
 
 /**
- * INativeAuthPublicClientApplication provides top level interface that is used by dpp developers
+ * INativeAuthPublicClientApplication provides top level interface that is used by app developers
  * to use Native Auth methods.
  */
 interface INativeAuthPublicClientApplication : IPublicClientApplication {
```

Comment:
Java docs needed for methods in these classes, as they are all customer facing. Can just copy the javadocs from `NativeAuthPublicClientApplication`

Reply:
I have added comment